src/mem/cache/base.cc

```
@@ -185,6 +185,8 @@ BaseCache::BaseCache(const BaseCacheParams &p, unsigned blk_size)
       stats(*this),
       cacheLevel(p.cache_level),
       forceHit(p.force_hit),
+      doFastWriteline(p.do_fast_writeline),
+      Prefetch_CanOffload(p.prefetch_can_offload)
```

Cppcheck (2.18.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Prefetch_CanOffload is a macro then please configure it.

(unknownMacro)